meshio/med_io.py

```diff
@@ -46,11 +47,29 @@ def read(filename):
         assert len(ts) == 1, "Must only contain exactly 1 time-step"
         mesh = mesh[list(ts)[0]]
 
+    # Read nodal and cell data if they exist
+    try:
+        cha = f["CHA"]  # champs (fields) in french
+        point_data, cell_data, field_data = _read_data(cha)
```

Comment:
Better: `try`, `except`,
```
else:
    point_data, cell_data, field_data = _read_data(cha)
```
(This way, we don't accidentally intercept `KeyError`s from `_read_data`.)